core/src/main/resources/lib/hudson/buildCaption.jelly

```diff
@@ -38,22 +38,12 @@ THE SOFTWARE.
     <div class="jenkins-app-bar__controls">
       <j:if test="${it.building}">
         <div class="build-caption-progress-container">
-          <table class="middle-align">
-            <tr>
-              <td>
-                ${%Progress}:
-              </td>
-              <td class="build-caption-progress-bar">
-                <t:buildProgressBar build="${it}" animate="true"/>
-              </td>
-              <td class="jenkins-table__link">
-                <j:if test="${it.parent.hasAbortPermission()}">
-                  <l:stopButton href="stop" confirm="${%confirm(it.fullDisplayName)}"
+          ${%Progress}:
+          <t:buildProgressBar build="${it}"/>
```

Comment:
the build cancel button is moved to a new line on my 13" MBP:

<img width="1079" alt="image" src="https://github.com/jenkinsci/jenkins/assets/21194782/f2884ac3-3152-4215-8d42-952d5438a8e0">

From master:

<img width="1071" alt="image" src="https://github.com/jenkinsci/jenkins/assets/21194782/018a24af-0195-4485-9149-b86078f7ebcf">

Reply:
Thanks, added `nowrap` to it.